GVFS/GVFS.UnitTests.Windows/Windows/Virtualization/WindowsFileSystemVirtualizerTests.cs

```diff
@@ -182,9 +185,9 @@ public void OnStartDirectoryEnumerationReturnsPendingWhenResultsNotInMemory()
 
                     Guid enumerationGuid = Guid.NewGuid();
                     gitIndexProjection.EnumerationInMemory = false;
-                    mockVirtualization.OnStartDirectoryEnumeration(1, enumerationGuid, "test").ShouldEqual(HResult.Pending);
+                    mockVirtualization.requiredCallbacks.StartDirectoryEnumerationCallback(1, enumerationGuid, "test", 1, "test").ShouldEqual(HResult.Pending);

```

Comment:
Pass in `TriggeringProcessId` and `TriggeringProcessImageFileName` here instead of `1, "test"`?